fastify.js

```diff
@@ -466,18 +466,40 @@ function build (options) {
     return it
   }
 
+  function _inject (server, opts, cb) {
+    if (cb) {
+      shot.inject(server, opts, cb)
+      return
+    }
+    return new Promise(resolve => {
+      shot.inject(server, opts, resolve)
+    })
+  }
```

Comment:
`_inject` is still defined here, can you remove the full function and inline calls to `shot.inject`  in `inject`?